app/src/main/java/org/oppia/app/ongoingtopiclist/OngoingTopicListFragmentPresenter.kt

```diff
@@ -0,0 +1,60 @@
+package org.oppia.app.ongoingtopiclist
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.appcompat.app.AppCompatActivity
+import androidx.fragment.app.Fragment
+import androidx.recyclerview.widget.LinearLayoutManager
+import org.oppia.app.databinding.OngoingTopicItemBinding
+import org.oppia.app.databinding.OngoingTopicListFragmentBinding
+import org.oppia.app.recyclerview.BindableAdapter
+import org.oppia.app.viewmodel.ViewModelProvider
+import javax.inject.Inject
+
+/** The presenter for [OngoingTopicListFragment]. */
+class OngoingTopicListFragmentPresenter @Inject constructor(
+  private val activity: AppCompatActivity,
+  private val fragment: Fragment,
+  private val viewModelProvider: ViewModelProvider<OngoingTopicListViewModel>
+) {
+
+  private lateinit var binding: OngoingTopicListFragmentBinding
+
+  fun handleCreateView(inflater: LayoutInflater, container: ViewGroup?, internalProfileId: Int): View? {
+    val viewModel = getOngoingTopicListViewModel()
+    binding = OngoingTopicListFragmentBinding.inflate(inflater, container, /* attachToRoot= */ false)
+    viewModel.setProfileId(internalProfileId)
+
+    binding.ongoingTopicListToolbar.setNavigationOnClickListener {
+      (activity as OngoingTopicListActivity).finish()
+    }
+
+    binding.ongoingTopicList.apply {
+      layoutManager = LinearLayoutManager(activity.applicationContext)
+      adapter = createRecyclerViewAdapter()
+    }
+
+    // NB: Both the view model and lifecycle owner must be set in order to correctly bind LiveData elements to
```

Comment:
Please make it multiline comment.

Reply:
It is already a  2 line comment also, this has been kept by Ben in this way in other files.